tables/no-no-g0.utb

```diff
@@ -96,13 +98,27 @@ comp6 Å 6-16
 #lowword - 36-36              # make double when hyphen surrounded by white spaces
 
 numsign 3456
-capsletter 6                    # single capital letter indicator
-begcapsword 6-6                  # a block of consecutive capital letters indicator
-noback endcapsword 56                   # TODO
+
+capsletter 6
+begcapsword 69-6
+noback endcapsword 569
 begcapsphrase 6-6-6
 lencapsphrase 2
 endcapsphrase after 6
 
+# variable #1 keeps track of when begcapsword has been suppressed
+noback pass2 _!$sp[@69-6]          ?#1=1   # suppress begcapsword when it is not the first character of the word
+noback pass2 [@69-6]$l.@569@234$sp @6-6    # suppress begcapsword if it needs to be terminated by a endcapsword,
+noback pass2 [@69-6]$l.@569@234~   @6-6    # unless the endcapsword is followed by a single "s"
+noback pass2 [@69-6]$l.@569        ?#1=1   #
+noback pass2 #1=1[]$l              @6      # insert capsletter before (uppercase) letter if begcapsword was suppressed
+noback pass2 #1=1@569              ?#1=0   # caps mode is terminated by endcapsword; suppress endcapsword if
+                                           # begcapsword was suppressed
+noback pass2 #1=1!$l               *#1=0   # caps mode is terminated by character that is not a letter
+
+# remove dot 9
+swapdd cleanvirtual 69,569 6,56
+noback pass4 %cleanvirtual %cleanvirtual
```

Comment:
Thank god you commented what you are doing here. Otherwise you'd win the price for most cryptic code sequence